### PR TITLE
docs: add comments for frrouting configure file

### PR DESCRIPTION
**Pull request by steveyiyo**

### CODE REVIEW

**Comment by ching-kuo** [comment]
@steveyiyo Can you move the PDF file one directory up.
`terrafrom` directory should only include Terraform stuffs
Thanks!

**Comment by steveyiyo** [comment]
@ching-kuo Did you mean README.md file?

**Comment by ching-kuo** [comment]
Yes

**Comment by steveyiyo** [comment]
@ching-kuo done, Could you please check it again?

**Comment by ching-kuo** [comment]
@steveyiyo Can you add how to run the Terraform script to setup the environment in the README
Please include

- Downloading/setting up OpenStack credential 
- Link to Terraform installation page 
- How to run Terraform script
- How to destroy the environment

Thanks

**Comment by steveyiyo** [comment]
OK, I will do it ASAP.

**Comment by steveyiyo** [comment]
@ching-kuo I have pushed the guide about how to run terraform scripts.

However, I prefer to create a new MarkDown file to introduce how to run terraform scripts. (It's with FRRouting Guide currently)
Once you merged to the master repo, I will open a new PR to do it.

Thanks!

File: self-paced-labs/network/frr_bgp/README.md [hunk 1]
```diff
@@ -396,4 +412,51 @@ replace:
...
+# 先初始化環境
+`terraform init`
+```
+
+接著，我們要設定 OpenStack 的環境資訊，修改並執行以下指令。
+
+```shell
+# 新增 OpenStack 資訊
+export OS_USERNAME="" # 輸入 OpenStack 的帳號
+export OS_TENANT_NAME="" # 輸入 OpenStack 的帳號
```
Comment (ching-kuo) [inline self-paced-labs/network/frr_bgp/README.md]: OS_TENANT_NAME is the project name in OpenStack

File: self-paced-labs/network/frr_bgp/README.md [hunk 2]
```diff
@@ -396,4 +412,51 @@ replace:
...
+export OS_PASSWORD="" # 輸入 OpenStack 的密碼
+export OS_AUTH_URL="https://openstack.cloudnative.tw:5000" # 輸入 OpenStack 的授權網址，可以在 API Access 中找到
```
Comment (ching-kuo) [inline self-paced-labs/network/frr_bgp/README.md]: We can download the credential file directly from dashboard, which will be easier in most cases.

File: self-paced-labs/network/frr_bgp/terraform/00_variables.tf [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Params file for variables
 
+# Configure the OpenStack Provider
+provider "openstack" {
+  region      = "RegionOne"
+}
+
```
Comment (ching-kuo) [inline self-paced-labs/network/frr_bgp/terraform/00_variables.tf]: Prefer not to hard code this, this should be included in the credential file.

**Review by ching-kuo — CHANGES_REQUESTED** [review]
Minor comments

**Comment by steveyiyo** [comment]
@ching-kuo Thanks for the feedback. Please check it again.

File: self-paced-labs/network/frr_bgp/README.md [hunk 4]
```diff
@@ -396,4 +412,48 @@ replace:
...
+
+# 會跳出確認視窗，請輸入 `yes` 來確認。
+```
+
+## How to destroy the environment
+
+當實驗結束後，我們可以使用以下指令來銷毀環境：
+
+```shell
+terraform apply -destroy
```
Comment (ching-kuo) [inline self-paced-labs/network/frr_bgp/README.md]: ```terraform destroy``` seems more common

**Review by ching-kuo — COMMENTED** [review]
Minor comment, otherwise LGTM

**Review by ching-kuo — APPROVED** [review]

**Pull request merged**